erpnext_com/www/docs/user/manual/en/accounts/accounting-dimensions.md

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
the new images are not proper, inconsistent width on one and the other needs cropping from the top